spec/specification/linter_spec.rb

```diff
@@ -462,10 +462,11 @@ def result_should_include(*values)
         result_should_include('script_phases', 'Missing required shell script phase options `script` in script phase `Hello World`.')
       end
 
-      it 'checks script phases include the required keys' do
+      it 'checks script phases does not include unknown keys' do
```

Comment:
this seems like there should be two tests that:

* Ensure required keys are specified
* Ensure all keys that are provided are known/valid

Reply:
its already added above

Reply:
ohh I see now 👍